### PR TITLE
Problem: Missing demo repo issues reporting guide (fix cronos-labs/play-unreal-plugin#88)

### PR DESCRIPTION
**Pull request by damoncro**

### CODE REVIEW

**Review by leejw51crypto — APPROVED** [review]
lgtm

**Comment by damoncro** [comment]
Done~ @leejw51crypto

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Code of Conduct
+
+## Conduct
+### Contact: https://airtable.com/shrFiQnLrcpeBp2lS
```
Comment (tomtau) [inline CODE_OF_CONDUCT.md:4]: this links to defi wallet's contact

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+2. Crypto.com will investigate and validate the security issue submitted to us as quickly as we can, usually within 10 business days of receipt. Submitting a thorough report with clear steps to recreate the vulnerability and/or a proof-of-concept will move the process along in a timely manner.
+3. Crypto.com will acknowledge the bug, and make the necessary code changes to patch it. Some issues may require more time than others to patch, but we will strive to patch each vulnerability as quickly as our resources and development process allow.
+4. Crypto.com will publicly release the security patch for the vulnerability, and acknowledge the security fix in the release notes once the issue has been resolved. Public release notes can reference to the person or people who reported the vulnerability, unless they wish to stay anonymous.
+
+## Contact Us
+If you find a security issue, you can report it on the [Crypto.com HackerOne Bug Bounty Program](https://hackerone.com/crypto) or you can contact our team directly at [oss@crypto.com](mailto:oss@crypto.com).
```
Comment (tomtau) [inline SECURITY.md:17]: the bounty doesn't cover cronos play

**Review by tomtau — CHANGES_REQUESTED** [review]
the code of conduct and security policy don't match this repo -- you can check with @ricky321q @ken-cronos for their formulations and contacts

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (damoncro) [reply CODE_OF_CONDUCT.md:4]: It is the same as:
https://github.com/cronos-labs/play-unreal-demo/blob/main/README.md?plain=1#L45
https://github.com/cronos-labs/play-unreal-plugin/blob/main/README.md?plain=1#L52
Should it change? @ricky321q

File: SECURITY.md (hunk 2 above)
Comment (damoncro) [reply SECURITY.md:17]: It is the same as https://github.com/cronos-labs/play-unreal-plugin/blob/main/SECURITY.md?plain=1#L17.

Should we remove it in play-unreal-plugin as well?

Comment (damoncro) [reply SECURITY.md:17]: @ricky321q

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ricky321q) [reply CODE_OF_CONDUCT.md:4]: I think this link (https://airtable.com/shrFiQnLrcpeBp2lS) is the right form to contact to us.
What do you mean for links to defi wallet's contact? @tomtau

File: SECURITY.md (hunk 2 above)
Comment (ricky321q) [reply SECURITY.md:17]: Yeah better remove it if it not covered. @damoncro

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (tomtau) [reply CODE_OF_CONDUCT.md:4]: oh, perhaps ok

File: SECURITY.md (hunk 2 above)
Comment (damoncro) [reply SECURITY.md]: Removed.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (tomtau) [reply CODE_OF_CONDUCT.md:4]: you can see below:

> Remarks that violate the Crypto.com standards of conduct, (...)

> In the Crypto.com developer community (...)

Comment (tomtau) [reply CODE_OF_CONDUCT.md:4]: maybe ok as is, it may be clear that the moderator is cronos labs

**Review by tomtau — APPROVED** [review]

**Pull request merged**